python/lsst/ts/vent/controller/dispatcher.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import traceback
+
+from lsst.ts import tcpip
+
+from .controller import Controller
+
+
+def _type_convert(t: type, value: str):
+    """Converts the value string to the specified type. In the case of boolean,
+    "True" or "T" or "1" is ``True`` and everything else is ``False``. Other
+    cases are handled by cast.
+
+    Parameters
+    ----------
+    t: type
+        The type that value should be converted to.
+
+    value: str
+        The value to be converted.
+
+    Raises
+    ------
+    ValueError
+        If the value cannot be converted to the specified type.
+    """
+
+    if t == bool:  # Boolean is a special case
```

Comment:
I think in this case you might want to use `is` instead of equality. Because what you are trying to determine here is if `t` is the same as a `bool`. 

I would suggest doing `if t is bool`.

Guidelines on comparisons: https://developer.lsst.io/python/style.html#comparisons